libs/packages/components/src/lib/tooltip/tooltip.directive.ts

```diff
@@ -0,0 +1,53 @@
+import { AfterViewInit, Directive, ElementRef, HostListener, Input, Renderer2, TemplateRef } from '@angular/core';
+
+@Directive({
+  selector: '[sdsTooltip]',
+  exportAs: 'sdsTooltip'
+})
+export class SdsTooltipDirective implements AfterViewInit {
+
+  private _sdsTooltip: string | TemplateRef<any> | HTMLDivElement;
+  sdsTooltipDiv: HTMLElement;
+
+  @Input()
+  position: string = 'top';
+
+  @HostListener('focus', ['$event']) onFocus(){
+    this.renderer.setAttribute(this.sdsTooltipDiv, 'aria-hidden', 'false')
+  }
+  @HostListener('blur', ['$event']) onBlue(){
```

Comment:
It likely doesn't effect functionality - small typo on the function name `onBlue`

Reply:
Fixed